CMakeLists.txt

```diff
@@ -406,24 +409,22 @@ else()
   option(ENABLE_JEMALLOC "enable jemalloc" ON)
 endif()
 
-if (ENABLE_JEMALLOC)
+if(ENABLE_JEMALLOC)
   if(CLANG_ASAN_UBSAN OR CLANG_MSAN OR CLANG_TSAN)
     message(STATUS "Sanitizers have been enabled; don't use jemalloc.")
   else()
-    find_package(JeMalloc)
-    if(JEMALLOC_FOUND)
-      include_directories(SYSTEM ${JEMALLOC_INCLUDE_DIRS})
-    endif()
+    find_package(JeMalloc REQUIRED)
+    include_directories(SYSTEM ${JEMALLOC_INCLUDE_DIRS})
   endif()
 endif()
 
-find_package(LibIntl)
-if(LibIntl_FOUND)
+if(ENABLE_LIBINTL)
+  find_package(Intl REQUIRED)
```

Comment:
@jamessan @jszakmeister Is `find_package(LibIntl)` needed instead of `find_package(Intl)` for some reason?

Reply:
Yes, because find_package(LibIntl) is actually delegating to our cmake/FindLibIntl.cmake file.  If we provide the wrong name, then CMake can't locate it.

Reply:
BTW, I'm not sure why `find_package(Intl)` seems to work and `find_package(LibIntl)` doesn't. :-/  Maybe it was partially related to gettext missing?  But I don't really see how that would happen.

Reply:
Upstream's is `FindIntl`:

```
[jamessan@freya] 0 % dpkg -S FindLibIntl
dpkg-query: no path found matching pattern *FindLibIntl*
[jamessan@freya] 1 % dpkg -S FindIntl
cmake-data: /usr/share/cmake-3.11/Modules/FindIntl.cmake
cmake-data: /usr/share/cmake-3.11/Help/module/FindIntl.rst
```

Is there a reason we don't use that or build our own on top of that?

Reply:
Oh, and one other thing.  The FindIntl.cmake that comes with CMake doesn't do the work of finding it via Homebrew--which is why we don't use the standard one.

Reply:
@jamessan Yes, it doesn't find it when Homebrew is involved on macOS.

Reply:
Isn't there a way we can instruct `FindIntl` to consider homebrew's paths too?